.github/workflows/compare_momepy.yaml

```diff
@@ -27,6 +31,9 @@ jobs:
 
       - name: Compare rcoins output to momepy output
         run: |
-          Rscript -e 'devtools::install()'
+          # install packaged data
+          Rscript -e 'devtools::install_github("CityRiverSpaces/CRiSpData")'
+          # install the current package
+          Rscript -e 'devtools::install(dependencies = FALSE)'
```

Comment:
why not install the dependencies? CRiSpData is already one of the `Suggests` dependencies, right? for example, we could do `devtools::install(dependencies = "Suggests", quiet = TRUE, upgrade = "always")`

Reply:
Worked! :)